modules/navigation.js

```diff
@@ -0,0 +1,37 @@
+const bookForm = document.getElementById('book-form');
+const bookLists = document.getElementById('book-section');
+const contact = document.getElementById('contact');
+
+const add = document.getElementById('add');
+const list = document.getElementById('list');
+const contactInfo = document.getElementById('contact-info');
+
+bookForm.style.display = 'flex';
+bookLists.style.display = 'none';
+contact.style.display = 'none';
```

Comment:
- [OPTIONAL] I think it would be better to put all your styles in CSS file for best practice purposes:recycle:

Reply:
@GedeonTS Cool cool. Got you. I will add it then.

Reply:
@GedeonTS I've discovered that the issue is connected to the Luxon Library. I want to highlight that the Luxon Library isn't present in this repository because it's excluded in the .gitignore file. The reason behind this is that the installation of the library takes place within the node_modules folder.

Here [Video](https://www.loom.com/share/dbf2c934ac524311945afdebf22fd76cl)